compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -520,7 +521,8 @@ setMatmulVectorDistributionConfig(mlir::FunctionOpInterface entryPoint,
   GPUMatmulShapeType problem{bounds[mDim], bounds[nDim], bounds[kDim],
                              lhsElemType,  rhsElemType,  initElemType};
 
-  auto mmaAttrs = llvm::to_vector(mmaKinds->getAsRange<IREE::GPU::MmaAttr>());
+  auto mmaAttrs =
+      llvm::to_vector(mmaKinds->getAsRange<IREE::GPU::MmaInterfaceAttr>());
```

Comment:
Somewhere in here this needs to be conditioned on subgroup_size == 32. We could add the subgroup size to the `MMAInterfaceAttr` and require that the subgroup size matches the intrinsic, or we could just add support for size == 64 and pass in subgroup size as an argument to `getContractionLayout`.

Reply:
Ah yeah good catch Quinn. I added the subgroup size to MMAInterfaceAttr and added the condition to match targetSubgroupSize and the layout subgroup size.

Since the layout for subgroupsize == 64 would be different than for subgroupsize == 32 though, so we'd need to add the subgroup size to the MMAInterfaceAttr anyways. I am thinking of adding the other WMMA layouts (including the subgroup_size==64) on a follow-on PR.